src/command_modules/azure-cli-ams/setup.py

```diff
@@ -33,7 +33,7 @@
 
 DEPENDENCIES = [
     'azure-cli-core',
-    'azure-mgmt-media==1.0.0rc1',
+    'azure-mgmt-media==1.0.0',
```

Comment:
Please update to 1.0.1, there is a package trouble in 1.0.0 that prevents installation of the CLI in Homebrew (@tjprescott )

Reply:
Agreed. You must update this to 1.0.1.